src/aurelia-slickgrid/extensions/slickRowDetailView.ts

```diff
@@ -275,11 +268,10 @@ export class SlickRowDetailView extends UniversalSlickRowDetailView {
   // ------------------
 
   protected disposeViewSlot(expandedView: CreatedView) {
-    if (expandedView && expandedView.view && expandedView.viewSlot && expandedView.view.unbind && expandedView.viewSlot.remove) {
+    if (expandedView && expandedView.controller) {
```

Comment:
ah it looks like I missed it earlier but we could use optional chaining here `?.`, it's also nice to see that it got way simpler in Au2 for dynamic components